homeassistant/components/zwave_me/translations/en.json

```diff
@@ -13,7 +13,7 @@
                     "token": "Token",
                     "url": "URL"
                 },
-                "description": "Input IP address of Z-Way server and Z-Way access token. IP address can be prefixed with wss:// if HTTPS should be used instead of HTTP. To get the token go to the Z-Way user interface > Menu > Settings > User > API token. It is suggested to create a new user for Home Assistant and grant access to devices you need to control from Home Assistant. It is also possible to use remote access via find.z-wave.me to connect a remote Z-Way. Input wss://find.z-wave.me in IP field and copy the token with Global scope (log-in to Z-Way via find.z-wave.me for this)."
+                "description": "Input IP address with port and access token of Z-Way server. To get the token go to the Z-Way user interface Smartome > Menu > Settings > Users > Administrator > API token. To grant access only to certain devices, create a new user and select the necessary devices from the list.\n\nExample of connecting to Z-Way on a local network:\nURL: {local_url}\nToken: {local_token}\n\nExample of connecting to Z-Way via remote access find.z-wave.me:\nURL: {find_url}\nToken: {find_token}\n\nExample of connecting to Z-Way with a static IP address:\nURL: {remote_url}\nToken: {local_token}\n\nIP address can be prefixed with wss:// if HTTPS should be used instead of HTTP. When connecting via find.z-wave.me you need to use a token with a global scope (log-in to Z-Way via find.z-wave.me for this)."
```

Comment:
You need to update the strings.json file. That's the source of truth.

Reply:
I suggest moving the details to the docs page instead to shorten the description here. Just keep the two first sentences and the three examples here.

Reply:
The user experience showed that people are frustrated. So we decided to improve the description with examples. As a compromise, we can do a link to the manual right in this doc. Is there a standard way to do it? But I would prefer the current way.

Reply:
So we change only strings.json. But is it possible to keep own translation to other languages still maintaining automatics translation for others?

Am I right that we commit only config_flow.py and strings.json and all translations we remove from the PR? How about RU translation? Can we keep it custom?

Reply:
We'll add a link to the docs in the next release by default. You don't need to add that explicitly.

Reply:
Please point me to a page where to add a detailed description

Reply:
In this case, should we remove the placeholder back? I'm still worried we don't include examples as customers are a bit frustrated where to find the key and what to put here

Reply:
For the third time: I think we should keep the examples in the form description. Only move the other details to the docs.

Reply:
Here's your docs page: https://www.home-assistant.io/integrations/zwave_me/